index.bs

```diff
@@ -47,10 +47,20 @@ spec: HTML51; urlPrefix: http://www.w3.org/TR/html51/; for: web
             text: opaque origin; url: opaque-origin; for:web
             text: tuple origin
             text: relaxing the same-origin restriction
+            text: domain; url: origin-domain
     type: dfn
         urlPrefix: webappapis.html;
             text: current settings object; for:web; url:current-settings-object
             text: Navigator; for: interface; url:the-navigator-object
+
+<!-- spec: HTML; urlPrefix: https://url.spec.whatwg.org/ -->
+spec: URL; urlPrefix: https://url.spec.whatwg.org/; for: url
```

Comment:
Most of these anchors should not be necessary. (Both here and for HTML, at least if you reference HTML instead of HTML51.)

In general it's best to minimize the number of anchors you have; it implies either the original spec is not correctly exporting its terms, or you're not using enough metadata to tell Bikeshed what's what (e.g. for="" attributes).